packages/npm/@amazeelabs/gatsby-starter/gatsby-node.ts

```diff
@@ -1,34 +1,20 @@
 // Because we used ts-node in gatsby-config.js, this file will automatically be
 // imported by Gatsby instead of gatsby-node.js.
 
-import { ForkTsCheckerWebpackPlugin } from 'fork-ts-checker-webpack-plugin/lib/ForkTsCheckerWebpackPlugin';
```

Comment:
Which tool is used to type-check now that `ForkTsCheckerWebpackPlugin` has been removed?

Reply:
Oh, just re-read the description and found out that type-checking is done before commit by `jest`.